pkg/cloud/externalconn/impl/external_connection.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 The Cockroach Authors.
```

Comment:
Renamed the file to `registry.go`. I didn't want to put all the imports in `impl_registry.go` because that then means everyone that wants the external connections registered has to pull in a dependency to the base `externalconn` package.